suite-common/token-definitions/README.md

```diff
@@ -0,0 +1,54 @@
+### Token definitions
```

Comment:
nitpick (followup?) - you need COINGECKO_API_KEY to be able to call the script, right? If so, would be nice to mention it in the README.

Reply:
548b988a94ba77b854e0e1a35396eccf88c35f5a